recipes/cylc-uiserver/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "cylc-uiserver" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4cc85a6680148cddd8f1fd714f772acb7c52487239b7c2b781bf543a51c099d1
+
+build:
+  number: 0
+  # cylc-uiserver provides JupyterHub extensions and a small Jupyter Application. As JupyterHub
+  # does not run on Windows, we skip the build for that platform here
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    # TODO: build failed with pytest-runner only, because the setup.py of cylc-uiserver specifies this
+    # version of pytest-runner. Need to make pytest-runner a dependency only for python setup.py test
+    # and then remove this from requirements|host in the recipe
+    - pytest-runner ==4.4
```

Just a small hint here, the ``==`` is not necessary :)
Not necessary to change, but you can just write
```suggestion
    - pytest-runner  4.4
```

---

Oh that works too? Good to know, looms simpler that way. Thanks!